### DISCUSSION TITLE
Assistance with GitHub Education Pack activation for Personal Email and Microsoft Azure access

### DISCUSSION THREAD

**Discussion by BryanBradfo** [community/community#118393] [GitHub Education]
### Hi everyone,

Dear GitHub Education Support Team,

I am writing to seek your assistance with an issue I am encountering related to the GitHub Education Pack, specifically in conjunction with activating Microsoft Azure for my academic pursuits.

Recently, I attempted to activate Microsoft Azure using my academic email addresses (@etu.toulouse-inp.fr and @etu.inp-n7.fr) for verification purposes. Unfortunately, according to the Microsoft Azure for Students Support 
Europe, Middle East and Africa, these addresses were not recognized by Microsoft's database as valid academic IDs, thereby preventing me from proceeding with the activation. Following a consultation with Microsoft's support team (as you can see below, but it is in French, sorry for the inconvenience), I was advised that my academic domains have not been validated within the Azure Dev Tools For Teaching/Azure for Students database as academic domains. They recommended that I first activate the GitHub Student Developer Pack using my personal email address and then proceed with the Azure for Students activation.

![img_of_mail](https://github.com/community/community/assets/101939095/51d2498a-ecbe-412e-a2f9-c162efe24d0f)

In light of this, I kindly request your guidance on how to activate the GitHub Student Developer Pack using my personal email address, as this seems to be a prerequisite for accessing Azure for Students via the GitHub option. I understand that normally, academic verification is done through institutional email addresses, but under the circumstances, I am compelled to use a personal one.

Moreover, Microsoft's support team indicated that should I encounter any difficulties after activating the GitHub Student Pack and attempting to access Azure for Students, I would need to provide:

1. A screenshot from the GitHub Education benefits portal showing my student status as approved.
2. Detailed screenshots or a step-by-step account of any errors or issues encountered during the Azure for Students subscription process, ideally documented using the Problem Steps Recorder (PSR) tool.

I am prepared to provide the necessary documentation as requested. However, I first need to ensure that the GitHub Student Developer Pack is activated on my personal email address. Could you please advise on how to proceed with this unique situation? Your guidance on how to navigate this process would be immensely appreciated.

If needed, I can provide my email address, but I prefer to give it in private if possible. 

Thank you very much for your time and assistance. I look forward to your advice on the steps I should take to resolve this matter and successfully access the resources provided by Microsoft Azure for my academic projects.

Warm regards,
BryanBradfo

**Comment by marybucy** [comment]
To activate the GitHub Education Pack for your personal email and gain access to Microsoft Azure benefits, follow these steps:

Create a GitHub Account: If you haven't already, sign up for a GitHub account using your personal email address.

Verify Your Student Status: Visit the GitHub Education website (education.github.com) and sign in with your GitHub account. Follow the prompts to verify your student status using a valid student ID, school-issued email, or other accepted forms of verification.
If anyone want to know about [commercial floor epoxy](https://www.evolveepoxy.com/) visit here

**Reply by BryanBradfo** [reply]
I already created my GitHub Education Pack and I am already validated by the Github Education Team. Actually, I even already have CoPilot. The thing is that when I try to activate Microsoft Azure using my academic email addresses (@etu.toulouse-inp.fr and @etu.inp-n7.fr), it is not working at all, and after contacting the Microsoft Azure for Students Support, Europe, Middle East and Africa Team, they say that I have to activate the GitHub Student Developer Pack using my personal email address, but it is activated with my academic email address. How to fix this issue?
